kolibri/plugins/learn/assets/src/views/TopicsPage.vue

```diff
@@ -210,6 +268,26 @@
         context: 'DO NOT TRANSLATE',
       },
       viewMore: 'View more',
+      folders: {
```

Comment:
Delete this and use the new string `Folders` that you would have created in the [`commonLearnStrings.js`](https://github.com/learningequality/kolibri/pull/8424/files#r707883099) 😉